pokedex/pokemon.lua

```diff
@@ -37,7 +37,7 @@ local feat_to_attribute = {
 	Acrobat="DEX"
 }
 
-local LATEST_POKEMON_VERSION = 2
+local LATEST_POKEMON_VERSION = 3
```

Comment:
This probably conflicts with the stuff I checked in for variants, so you may need to update this to 4 after merging from master.